vmdb/spec/views/ems_infra/_form_fields.html.haml_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "spec_helper"
+
+describe "rendering fields in ems_infra new/edit form" do
+  before(:each) do
+    @edit = {:new => {:emstype => "vm_ware"}}
+  end
+
+  it "displays Host Name" do
+    render :partial => "ems_infra/form_fields", :locals => {:url => ""}
+    expect(rendered).to match /Host\ Name/
+  end
+
+  it "doesn't display IP Address" do
+    render :partial => "ems_infra/form_fields", :locals => {:url => ""}
+    expect(rendered).not_to match /\AIP\ Address/
```

Comment:
Thanks for writing the specs.

Is it possible to loop through the expected `td.key` values from the render call to ensure we have N number of cells and the keys we expect are there?  I don't know.

In the long run, I'm not sure how much value there is to checking that IP Address is not found.

cc @matthewd @tenderlove any ideas on how to test renders like this?